bundles/org.openhab.binding.magentatv/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -5,5 +5,6 @@
 
 	<name>MagentaTV Binding</name>
 	<description>This is the binding for MagentaTV receivers</description>
+	<author>Markus Michels</author>
```

Comment:
The author tag is deprecated, we don't use it anymore.
```suggestion
```

Reply:
removed